asciidoc/modules/con_mg_migrating-applications-to-new-redis-client.adoc

```diff
@@ -0,0 +1,187 @@
+[id="migrating-applications-to-new-redis-client_{context}"]
+= Migrating existing Redis client applications to new client
+
+You can migrate your existing applications to new `Redis` client directly or use the helper class `RedisAPI` to move your in two steps.
+
+Before migrating the applications you must create the client.
+
+== Creating the client
+
+The following example shows how a Redis client was created in {VertX} {3x} releases.
+
+----
+// Create the redis client (3.x)
+RedisClient client = RedisClient
+  .create(vertx, new RedisOptions().setHost(host));
+----
+
+The following example shows how to create a Redis client in {VertX} {4}.
+
+----
+// Create the redis client (4.x)
+Redis client = Redis
+  .createClient(
+    vertx,
+    "redis://server.address:port");
+----
+
+In {VertX} {4}, the client uses the following standard connection string syntax:
+
+----
+redis[s]://[[user]:password@]server[:port]/[database]
+----
+
+== Migrating applications to `RedisAPI`
+
+Using the 'ResdisAPI` you can now decide how to manage the connection:
```

Comment:
typpo: `ResdisAPI` should be `RedisAPI`

Reply:
Done. Thank you for catching this @pmlopes :)